api/webhook/verify.go

```diff
@@ -0,0 +1,49 @@
+package webhook
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+func verifyRequestSignature(r *http.Request) ([]byte, error) {
+	ctx := shared.NewAppEngineContext(r)
+	log := shared.GetLogger(ctx)
+
+	payload, err := ioutil.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		log.Errorf("Failed to read request body: %s", err.Error())
+		return nil, errors.New("Failed to read request body")
+	}
+
+	secret, err := getSecret(ctx)
```

Comment:
We should move `getSecret` to this file as well. Besides, it might be worthwhile to generalize the `getSecret` (and this function as well) to take in the name for the token, so that we can use different tokens in different webhooks.

Reply:
Agreed (and actually already in flight) - copy-pasted the changes from https://github.com/web-platform-tests/wpt.fyi/pull/732/files#diff-04b8cc3bdfca7285a8fb0ee07b8069ccR283